test/scripts/hexo/post.js

```diff
@@ -15,7 +15,26 @@ describe('Post', () => {
   const { post } = hexo;
   const now = Date.now();
   let clock;
-  const defaultCfg = JSON.parse(JSON.stringify(hexo.config));
+  const defaultCfg = JSON.parse(JSON.stringify(Object.assign(hexo.config, {
+    marked: {
```

Comment:
AFAIK, `hexo-renderer-marked` could handle default configuration. Why add `marked` here?

Reply:
It's a workaround, Actions [failed](https://github.com/hexojs/hexo/runs/1044635222) without it, but it works in my machine though.

Reply:
Utilise `afterEach()`.